src/main/resources/changesets/execution_backend_info.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="CREATE_EXECUTION_INFO">
+        <createTable tableName="EXECUTION_INFO">
```

Comment:
The other problem this model creates is that how does the engine will know the specifics for each backend model?